frontend-react/src/hooks/UseValueSets.test.ts

```diff
@@ -89,9 +89,10 @@ describe("useValueSetUpdate", () => {
 
     test("returns trigger and loading indicator", async () => {
         const { result } = renderWithQueryWrapper();
-        const { saveData, isSaving } = result.current;
+        const { saveData, isSaving, saveError } = result.current;
         expect(isSaving).toEqual(false);
         expect(saveData).toBeInstanceOf(Function);
+        expect(saveError).toBeNull();
```

Comment:
We have, and can use, this error from mutation hooks, but because of how we handle the UI for mutation failures, we'll likely rarely use it. Regardless, I added it back so we have access to it, and tests are written to ensure we have that access over the evolution of the hook.